src/accordion/stories/AccordionBasic.component.tsx

```diff
@@ -5,10 +5,10 @@ import {
   AccordionPanel,
   AccordionTrigger,
   useAccordionState,
-  AccordionInitialState,
 } from "renderless-components";
+import { AccordionInitialState } from "../types";
```

Comment:
Not Needed !!

Reply:
i've currently set any in the App prop... so will add it back since storybook is forcing the allowMultiple to be true

Reply:
Oh, right right !!